_data/sites.json

```
@@ -5314,6 +5314,16 @@
         ]
     },
 
+    {
+        "name": "Postman",
+        "url": "https://app.getpostman.com/dashboard/profile",
+        "difficulty": "easy",
+        "notes": "Log into your account -> Click the link -> Click the Delete Account.",
```

I'm a little unclear on what "Click the link" is meant to refer to--is there only one link on the page after logging in?

---

No i mean referring link. is it unnecessary?

---

Oh, so you're just referring to the link which would be shown on justdeleteme.xyz? In that case, I think this would work better as simply:
```suggestion
        "notes": "Log into your account, then click the 'Delete Account' button on your profile page.",
```

(or "link" if the thing to click on isn't a button)

---

Yes that's better.